cubedash/_model.py

```diff
@@ -7,6 +8,7 @@
 import flask_themes
 import structlog
 from flask_caching import Cache
+from prometheus_flask_exporter.multiprocess import GunicornInternalPrometheusMetrics
```

Comment:
It's listed as an optional dependency, so the import should be optional too. Perhaps do the same thing as the raven import.